Csharp/sigmf/sigmf/SigMFReader.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace sigmf
+{
+    public class SigMFReader
+    {
+        public static MetaDoc read(string fileName) {
+            MetaDoc meta = JsonConvert.DeserializeObject<MetaDoc>(File.ReadAllText(fileName));
```

Comment:
I don't think this works yet. It produces a MetaDoc, but it's empty. I suspect this is due to the problem in my other comment, but I would test this component further just to ensure we can properly deserialize SigMF produced by sensors.